src/lib/get-configuration/get-configuration.test.js

```diff
@@ -168,9 +170,13 @@ describe('config', () => {
     };
 
     expect(() => getConfiguration({ constants, inputs })).toThrow(
-      new Error(
-        `Invalid JSON for 'thresholds' input: Unexpected token i in JSON at position 0`,
-      ),
+      getNodeMajorVersion() >= 19
+        ? new Error(
+            "Invalid JSON for 'thresholds' input: Unexpected token 'i', \"invalid_json\" is not valid JSON",
+          )
+        : new Error(
+            `Invalid JSON for 'thresholds' input: Unexpected token i in JSON at position 0`,
+          ),
```

Comment:
Instead of conditionally checking the node version and creating an error to match against, I think we could do:

```js
expect(() => getConfiguration({ constants, inputs })).toThrow(/Invalid JSON for 'thresholds' input/);
```

I feel like this is sufficient in terms of testing as we'd be verifying that any version of node is getting the important part of the error string

What do you think?

Reply:
I tried to keep it as close to the original as possible, but I agree that your solution is much more elegant. I will make an update containing this!